modules/client-configuration/pages/clients-oracle.adoc

```diff
@@ -93,7 +93,7 @@ oraclelinux8-appstream oraclelinux8-uyuni-client
 The client tools channel provided by [command]``spacewalk-common-channels`` is sourced from {uyuni} and not from {suse}.
 ====
 
- .Procedure: Synchronizing Oracle repositories
+.Procedure: Synchronizing Oracle repositories
  . In the Uyuni Web UI, navigate to Software Manage, and check every Oracle channel.
  . In the Repositories tab, navigate to the Sync subtab, and click Sync Now. You can also create a regular synchronization schedule on this page.
```

Comment:
here and above we want adoc tag soup ;)

Reply:
Yeah, this will all get changed when I fix these chapters, though, so I'm not too concerned at this point.